src/canmatrix/canmatrix.py

```diff
@@ -338,27 +396,40 @@ def raw2phys(self, value, decodeToStr=False):
     def __str__(self):
         return self.name
 
+
 class SignalGroup(object):
     """
-    contains Signals, which belong to signal-group
+    Represents signal-group, containing multiple Signals.
     """
 
     def __init__(self, name, Id):
-        self.signals = []
+        self.signals = []  # use attr.ib?
         self.name = name
         self.id = Id
 
     def addSignal(self, signal):
+        """Add a Signal to SignalGroup.
+
+        :param Signal signal: signal to add
+        """
         if signal not in self.signals:
             self.signals.append(signal)
 
     def delSignal(self, signal):
+        """Remove Signal from SignalGroup.
+
+        :param Signal signal: signal to remove
+        """
         if signal in self.signals:
-            self.signals[signal].remove()
+            self.signals.remove(signal)
 
-    def byName(self, name):
+    def byName(self, name):  # add 'Signal' to function name?
```

Comment:
It's a signal group, what would you get other than signals?  I mean, maybe, but that's my gut response.  :]

Reply:
But than, in code, you see `return tmp.byName(name)` and you're lost... The __getitem__() might be aslo useful. Postpone to another PR.. maybe

Reply:
I wouldn't use method names to help callers keep track of what sort of object they have.  It seems like the name referring to the object itself is what should be sufficiently descriptive for the context.